app/src/main/java/com/amaze/filemanager/filesystem/FileUtil.java

```diff
@@ -668,9 +669,11 @@ public static boolean isWritable(final File file) {
             try {
                 output.close();
             } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Is it a good idea not to do `Exception.printStackTrace()` here?

Reply:
What's the difference?

Reply:
I don't know the usage of this code apart from a call here. So I think it's better to print the whole stack trace in case operation don't complete (unlike here, when it was completed).